conjure-go-client/httpclient/internal/refreshingclient/transport.go

```diff
@@ -34,38 +34,29 @@ type TransportParams struct {
 	ExpectContinueTimeout time.Duration
 	ResponseHeaderTimeout time.Duration
 	TLSHandshakeTimeout   time.Duration
-	HTTPProxyURL          *url.URL `refreshables:",exclude"`
```

Comment:
To clarify, struct tag was for v1 codegen, so we don't need it now that we're not doing that?